Item Pickup Log

DidiSkywalker opened the pull request:
Adds an overlay element that displays changes in the inventory, like picking up or using items.
I'm creating this PR to document this feature as being worked on and discuss some elements, while waiting for refactoring to finish.
1. Currently detects shift-clicking items in the inventory as changes
2. Should the amount of time items are displayed for be customizable in the settings?

*Example from mining a Shiny Block*
![2019-08-08_19 54 54](https://user-images.githubusercontent.com/31072807/62746590-ca8e8a80-ba50-11e9-8421-532825cb3f7a.png)

ViralTaco commented:
I know I'm not helping but I have feedback on this: 

> 1. Currently detects shift-clicking items in the inventory as changes
> 2. Should the amount of time items are displayed for be customizable in the settings?

1. Check if inventory is opened
2. Should be chat default imo (assuming you used chat for that; didn't read the code yet) otherwise, assuming this gets merged, I'd say: yes

DidiSkywalker commented:
**Small changelog:**
* Moving items inside the inventory no longer display as changes
* Ignoring Quiver Arrow
* Increased display time to 5 seconds
* Merged current master into the branch
* Moved inventory related utility methods into new `InventoryUtils` class
* Added `.gitignore`

This should be all ready and mergable without conflicts now.

biscuut commented:
Wow thanks for this!
